canonical-web-and-design/jaas-dashboard
# Use a z-index scale

## Done

- Use a z-index scale
- Mapped values to the greek alphabet? Why the greek alphabet? Because it's arbitrary and doesn't couple the scale to the elements they are being applied to.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check that all page elements look correct across the board

## Details

Fixes: #112
